collection_size.go

```diff
@@ -22,6 +22,97 @@ type stats struct {
 	Size  int32 `bson:"size"`
 }
 
+// NewDBCollectionsSizeCollector returns a new collector that monitors sizes and object counts
+// of all collections in the specified database.
+func NewDBCollectionsSizeCollector(namespace string, db *mgo.Database) *dbCollectionsSizeCollector {
```

Comment:
I don't care enough to block landing it, But you need two things to make a database, session and a name string. But we never actually use it. That means clients have to make one - but it's never used: https://github.com/cloud-green/monitoring/pull/9/files#diff-be82fd1dce085c2ac204051ff78a9000R27

I'll let your conscience decide